.github/workflows/test.yml

```diff
@@ -0,0 +1,24 @@
+name: TDR Run Flyway Test
```

Comment:
Is there a reason this is called `TDR Run Flyway Test`?

Reply:
Because I copied it of the consignment-api-data repo and didn't change it :smile: I've renamed it.